modules/nodes-pods-vertical-autoscaler-configuring.adoc

```diff
@@ -8,6 +8,78 @@
 
 You can use the Vertical Pod Autoscaler Operator (VPA) by creating a VPA custom resource (CR). The CR indicates which pods it should analyze and determines the actions the VPA should take with those pods.
 
+////
+Hiding for now; working out details. **DO NOT REVIEW**
+If you are using the VPA on a custom resource that manages pods, the VPA requires that the `CustomResourceDefinition` object of the Custom Resource implements the `/scale` subresource with the optional `labelSelectorPath` field. The `labelSelectorPath` defines the JSON path inside the custom resource that corresponds to `Scale.Status.Selector`. The field pointed to by this JSON path must be a serialized label selector in string form. The following is an example of a `CustomResourceDefinition` and a `CustomResource` that fulfills these requirements, along with a `VerticalPodAutoscaler` definition that targets the custom resource. Note that this is an example to show the `/scale` subresource contract. Using the VPA on this example will not actually scale any pods because there is no controller that controls the custom resource to allow it to own any Pods.
```

Comment:
I think this paragraph is spot on and looks good, thanks.

Since we've QE'd this feature, is it okay if we uncomment this and the admission burst/qps table?